pkg/web/routes.go

```diff
@@ -103,6 +103,7 @@ func (s *Server) setupRoutes() (err error) {
 	s.router.GET("/counterparty", authenticate, s.CounterpartyVasps)
 	s.router.GET("/send-envelope", authenticate, s.SendEnvelopeForm)
 	s.router.GET("/utilities/travel-address", authenticate, s.TravelAddressUtility)
+	s.router.GET("/user-settings", authenticate, s.UserSettings)
```

Comment:
To match the case of the rest of the routes, I recommend it be `/users` -- although by this logic it should also be `/counterparties` above, but we can skip that one. 

I'll make that change if that's ok.